qsirecon/workflows/recon/amico.py

```diff
@@ -112,7 +100,9 @@ def init_amico_noddi_fit_wf(
         run_without_submitting=True,
     )
     noddi_fit = pe.Node(NODDI(**params), name="recon_noddi", n_procs=omp_nthreads)
+    desc += build_documentation(noddi_fit) + " "
     noddi_tissue_fraction = pe.Node(NODDITissueFraction(), name="noddi_tissue_fraction")
+    desc += "ICVF and Orientation Dispersion maps were multipled by the tissue fraction (1 - ISOVF) to produce tissue fraction modulated maps (@parker2021not)."
```

Comment:
This step just calculates the tissue fraction. The modulated maps seem to come from NODDI.

Reply:
So where do you want that description then?

Reply:
I think in the first desc (under `#### NODDI Reconstruction`). Maybe the description here should be something like "The tissue fraction (1 - ISOVF) was separately reconstructed with custom code."?